xml/System.Drawing.Imaging.Effects/Effect.xml

```diff
@@ -47,12 +47,12 @@
       </Docs>
     </Member>
     <Member MemberName="Dispose">
-      <MemberSignature Language="C#" Value="public virtual void Dispose (bool disposing);" />
-      <MemberSignature Language="ILAsm" Value=".method public hidebysig newslot virtual instance void Dispose(bool disposing) cil managed" />
+      <MemberSignature Language="C#" Value="protected virtual void Dispose (bool disposing);" />
+      <MemberSignature Language="ILAsm" Value=".method familyinstance hidebysig newslot virtual instance void Dispose(bool disposing) cil managed" />
       <MemberSignature Language="DocId" Value="M:System.Drawing.Imaging.Effects.Effect.Dispose(System.Boolean)" />
-      <MemberSignature Language="VB.NET" Value="Public Overridable Sub Dispose (disposing As Boolean)" />
+      <MemberSignature Language="VB.NET" Value="Protected Overridable Sub Dispose (disposing As Boolean)" />
       <MemberSignature Language="F#" Value="abstract member Dispose : bool -&gt; unit&#xA;override this.Dispose : bool -&gt; unit" Usage="effect.Dispose disposing" />
-      <MemberSignature Language="C++ CLI" Value="public:&#xA; virtual void Dispose(bool disposing);" />
+      <MemberSignature Language="C++ CLI" Value="protected:&#xA; virtual void Dispose(bool disposing);" />
```

Comment:
This should not be public and was made protected via https://github.com/dotnet/winforms/pull/12090

Reply:
These modifiers represent the RC1 API structure. I assume the above PR to make them protected was added in RC2?

We need to revert changes to these xml elements, as we are not allowed to change them manually. They need to be auto updated when we consume a new release.

Reply:
Gotcha, I'll revert this part of the change. This was changed to protected in RC2